modules/ROOT/pages/changes-deprecations-removals.adoc

```diff
@@ -0,0 +1,289 @@
+
+|link:{neo4j-docs-base-uri}/operations-manual/5/configuration/configuration-settings#config_server.memory.off_heap.block_cache_size[`server.memory.off_heap.block_cache_size`]
+|
+
+|link:{neo4j-docs-base-uri}/operations-manual/5/configuration/configuration-settings#config_server.memory.off_heap.max_cacheable_block_size[`server.memory.off_heap.max_cacheable_block_size`]
+|
+
+|link:{neo4j-docs-base-uri}/operations-manual/5/configuration/configuration-settings#config_server.memory.off_heap.transaction_max_size[`server.memory.off_heap.transaction_max_size`]
+|
+
+|===
+====
+
+
+[role=label--enterprise]
+=== Removed Neo4j metrics
+
+.See all removed metrics in Neo4j 2025.01
+[%collapsible]
+====
+[options="header"]
+|===
+|Name
+
+a|**link:{neo4j-docs-base-uri}/operations-manual/5/monitoring/metrics/reference/#raft-core-metrics[Raft core metrics] - replaced accordingly by the link:{neo4j-docs-base-uri}/operations-manual/5/monitoring/metrics/reference/#raft-metrics[Raft metrics]**
+
+|<prefix>.causal_clustering.core.append_index
+
+|<prefix>.causal_clustering.core.commit_index
+
+|<prefix>.causal_clustering.core.applied_index
+
+|<prefix>.causal_clustering.core.term
+
+|<prefix>.causal_clustering.core.tx_retries
+
+|<prefix>.causal_clustering.core.is_leader
+
+|<prefix>.causal_clustering.core.in_flight_cache.total_bytes
+
+|<prefix>.causal_clustering.core.in_flight_cache.max_bytes
+
+|<prefix>.causal_clustering.core.in_flight_cache.element_count
+
+|<prefix>.causal_clustering.core.in_flight_cache.max_elements
+
+|<prefix>.causal_clustering.core.in_flight_cache.hits
+
+|<prefix>.causal_clustering.core.in_flight_cache.misses
+
+|<prefix>.causal_clustering.core.raft_log_entry_prefetch_buffer.lag
+
+|<prefix>.causal_clustering.core.raft_log_entry_prefetch_buffer.bytes
+
+|<prefix>.causal_clustering.core.raft_log_entry_prefetch_buffer.size
+
+|<prefix>.causal_clustering.core.raft_log_entry_prefetch_buffer.async_put
+
+|<prefix>.causal_clustering.core.raft_log_entry_prefetch_buffer.sync_put
+
+|<prefix>.causal_clustering.core.message_processing_delay
+
+|<prefix>.causal_clustering.core.message_processing_timer
+
+|<prefix>.causal_clustering.core.replication_new
+
+|<prefix>.causal_clustering.core.replication_attempt
+
+|<prefix>.causal_clustering.core.replication_fail
+
+|<prefix>.causal_clustering.core.replication_maybe
+
+|<prefix>.causal_clustering.core.replication_success
+
+|<prefix>.causal_clustering.core.last_leader_message
+
+a|**link:{neo4j-docs-base-uri}/operations-manual/5/monitoring/metrics/reference/#read-replica-metrics[Read Replica metrics] - replaced accordingly by the link:{neo4j-docs-base-uri}/operations-manual/5/monitoring/metrics/reference/#store-copy-metrics[Store copy metrics]**
+
+|<prefix>.causal_clustering.read_replica.pull_updates
+
+|<prefix>.causal_clustering.read_replica.pull_update_highest_tx_id_requested
+
+|<prefix>.causal_clustering.read_replica.pull_update_highest_tx_id_received
+
+|===
+====
+
+=== SSL framework
```

Comment:
I'm not sure about this section. We can rename it or hide it in meantime

Reply:
I think it's fine.